src/css/mapbox-gl.css

```diff
@@ -769,3 +769,27 @@ a.mapboxgl-ctrl-logo.mapboxgl-compact {
         display: none;
     }
 }
+
+.mapboxgl-scroll-zoom-blocker-control {
+    color: #fff;
+    font-family: "Roboto", Arial, sans-serif;
```

Comment:
Do we ship GL JS with Roboto, or is this the system font stack?

If possible I'd love to have this using SF Text on macOS/iOS, and Ubuntu etc on Linux distros. I found a font stack here that I wonder if we should consider: https://markdotto.com/2018/02/07/github-system-fonts/

`font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;`

We could add Roboto to that too

Reply:
Thanks for the suggestion! I definitely want the font family to be consistent and be most user friendly!